### PR TITLE
Add chart maker, rename functions and other small things

### PR DESCRIPTION
**Pull request by JackBuckley**
## Do the following before requesting a review

- [ ] I have written tests covering functions I have added or changed.
- [ ] I have run tests and they all pass.
- [ ] I have ensured all new functions will show up in the `_pkgdown.yml` file.
- [ ] I have updated `DESCRIPTION` with any new package dependencies.
- [ ] I have updated `NEWS.md` with a brief description of my changes.
- [ ] I have updated the package documentation with `devtools::document()`.
- [ ] I have updated the package website with `pkgdown::build_site_github_pages()`.

### CODE REVIEW

File: R/e61_chart_maker.R [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#' @importFrom shiny dialogViewer browserViewer runGadget paneViewer reactiveValues
+#'
+#' @examples
+#' # Launch e61_chart_maker in RStudio :
+#' e61_chart_maker(iris)
+#'
+#' # Launch e61_chart_maker in without data specified :
+#' e61_chart_maker()
+
+e61_chart_maker <-
```
Comment (aaronw22) [inline R/e61_chart_maker.R:26]: New functions need to be added to the `_pkgdown.yml` (which I've done for you this time).

File: R/theme61.R [hunk 2]
```diff
@@ -241,9 +245,59 @@ theme_e61 <- function(base_size = 14,
...
+#' @import ggplot2 ggthemes
+#' @export
+#'
+#' @examples
+#' ggplot(data = mtcars, aes(x = wt, y = mpg, col = factor(cyl))) +
+#' geom_point() +
+#' e61_colour_manual(n = 3) +
+#' theme_e61_clean()
 
+theme_e61_clean <- function(
```
Comment (aaronw22) [inline R/theme61.R:266]: New functions need to be added to the `_pkgdown.yml` (which I've done for you this time).

**Review by elyse-22 — APPROVED** [review]

File: NEWS.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+# theme61 0.2.0
+
+* *Jack please add your changes here.*
```
Comment (aaronw22) [inline NEWS.md:3]: @JackBuckley please add your changes to the news file

**Review by aaronw22 — CHANGES_REQUESTED** [review]
Just need to tick off the things in the PR checklist and comments.

Would you also be able to try the deprecated functions and the renamed functions and see if they produce the appropriate warnings.

**Review by elyse-22 — APPROVED** [review]

**Review by aaronw22 — APPROVED** [review]

**Pull request merged**